core/pythonAction/cli/wskactivation.py

```diff
@@ -21,7 +21,13 @@
 from datetime import datetime, timedelta
 import copy
 from wskitem import Item
-from wskutil import addAuthenticatedCommand, apiBase, bold, parseQName, getQName, request, responseError, getPrettyJson
```

Comment:
all this source in pythonAction/cli we need to delete (these changes can may be replayed against the cli repo).
what to just delete this source (@mrutkows  fyi)